### ISSUE TITLE
Error in zypper ref on two days in a row??  Unable to update the system??

### ISSUE THREAD

**Issue by esteelpaz** [geckolinux/geckolinux-project#314]
@geckolinux 

Two days in a row my Gecko MATE edition has errored out on a zypper ref . . . did not find this issue in my TW install or Gecko Plasma editions . . . ???  So system will not upgrade, etc.

`Repository 'Tumbleweed_OSS' is invalid.
[Tumbleweed_OSS|http://download.opensuse.org/tumbleweed/repo/oss/] Valid metadata not found at specified URL
History:
 - [|] Error trying to read from 'http://download.opensuse.org/tumbleweed/repo/oss/'
 - Download (curl) error for 'http://download.opensuse.org/tumbleweed/repo/oss/repodata/repomd.xml':
Error code: Connection failed
Error message: Could not resolve host: download.opensuse.org
`

**Comment by geckolinux** [comment]
Thanks for reporting, yep, they seem to be having mirror issues:
https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/thread/NAJYGXBE6X4GB7WT37RLGUWCZDXG25R2/

**Comment by esteelpaz** [comment]
@geckolinux 

My pleasure to report the problem . . . .  I just rebooted out of Gecko rolliing and into reg TW and ran a zypper and . . . no issues??

`Retrieving repository 'Main Repository (NON-OSS)' metadata ...............[done]
Building repository 'Main Repository (NON-OSS)' cache ....................[done]
Retrieving repository 'Main Repository (OSS)' metadata ...................[done]
Building repository 'Main Repository (OSS)' cache ........................[done]
Repository 'Main Update Repository' is up to date.                              
Repository 'libdvdcss' is up to date.                                           
Retrieving repository 'Packman Repository' metadata ......................[done]
Building repository 'Packman Repository' cache ...........................[done]
All repositories have been refreshed.`

**Comment by geckolinux** [comment]
Yep, it's very random, their mirror server bounces you a different mirror and not all mirrors are apparently experiencing the same problem. At any rate these transient errors usually get resolved within 24h.

**Comment by esteelpaz** [comment]
@geckolinux 

OK, yes, that was "two days" worth of transiency . . . but, after running the TW upgrades an hour or so back I just now rebooted back into Gecko rolling and tried it again and . . . repos "passed muster" and upgrading is happening as we speak.

**Comment by geckolinux** [comment]
Thanks for confirming that!
@esteelpaz You seem to have a knack for discovering bugs. ;-) Would you be interested in testing the candidates for the next GeckoLinux updates?
https://sourceforge.net/projects/geckolinux/files/Rolling/staging/
There are some _major_ improvements such as an improved Btrfs configuration that permits rollbacks and bootable GRUB snapshots, as well as PipeWire with a low-latency audio config.

**Comment by esteelpaz** [comment]
@geckolinux 

Sorry for the delay . . . got busy . . . .  I was trying to change the domain name on one of my web sites and instead it looks like I blew it up . . . so I went down the rabbit hole on that one for a while . . . .  : - 0

So, back to the matter at hand . . . thanks for the opportunity . . . I'm somewhat busy, trying to stay "viable" in LA while also keeping 8 linux distros going in my desktop and several in two other laptops.  What are you asking??  Burn an iso to a flash drive and try to boot it up and run it???

I have two Gecko rolling installs in my '12 Mac Pro . . . I generally boot one system a day and rotate through the list for the week . . . .  My Gecko installs are formatted ext4 as per the advice on the SUSE forums back when I first started up with SUSE, so I wouldn't need to mess with btrfs (sic) . . . .

You have a nice system, if there is something specific I could do or check for you I'd be happy to fit it in . . . possibly after the holidays???  I don't think I'd want to do another bare metal install just yet.  Now I have to find and rebuild my errant web site, and then I've been stuck in writer's block on my third book since '14 and I'm just getting "unblocked" on it . . . .  If I can squeeze something in, like burning an iso and booting it I could do that in between all of my angst . . . .

I'm mostly just an end user who has several multi-boot machines, other than that I don't know what I'm doing.  I've fiddled with the Lubuntu devel list serve a bit, but they have all of these formal bureaucracies to go through to do a test which includes filing bug reports on each of the issues . . . .  I just don't have time to do that, I prefer to just let them know if I hit a problem via email.  Old guy on medicare . . . cranky. . .  and so forth.
